src/NuGetGallery.Services/Configuration/TrustedImageDomains.cs

```diff
@@ -38,7 +40,42 @@ public bool IsImageDomainTrusted(string imageDomain)
                 return false;
             }
 
-            return TrustedImageDomainList.Contains(imageDomain);
+            return ExpandedTrustedImageDomainList.Contains(imageDomain);
+        }
+
+        public HashSet<string> expandDomainList()
+        {
+            var expandedImageDomainList = new HashSet<string>();
```

Comment:
Should this be case insensitive? 

```suggestion
            var expandedImageDomainList = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
```